rt_dashboard/javascript/src/components/dashboard/dashboard.js

```diff
@@ -1,13 +1,17 @@
 import templateHtml from './dashboard.html'
 import { loadTemplate } from '../../utils/dom'
 
+const REFRESH_INTERVAL = 3000
+
 export default class Dashboard extends HTMLElement {
   constructor () {
     super()
 
     loadTemplate(this.attachShadow({ mode: 'open' }), templateHtml)
 
     this.selectedQueueChange = this.selectedQueueChange.bind(this)
+
+    setInterval(this.refreshInterval.bind(this), REFRESH_INTERVAL)
```

Comment:
Might be a good idea to have option to disable the interval - e.g. value of `0` or `-1`

Reply:
the interval is not cleared when component is removed